scheduler/src/cook/mesos/api.clj

```diff
@@ -740,6 +766,13 @@
                   "success" "failed")
                 :job.state/running "running"
                 :job.state/waiting "waiting")
+        constraints (->> job
+                         :job/constraint
+                         (map util/remove-datomic-namespacing)
+                         (map (fn [{:keys [attribute operator pattern]}]
+                                (->> [attribute (clojure.string/upper-case (name (util/without-ns operator))) pattern]
+                                     (remove nil?)
```

Comment:
The pattern is optional in the schema. I can make it required for now since the only operator needs it. Would you prefer that?